src/node/ext/ldap/base.py

```diff
@@ -41,7 +41,10 @@ def md5digest(key):
     :param key: Key to create a md5 hex digest for.
     :return digest: hex digest.
     """
-    m = hashlib.md5()
+    try:
+        m = hashlib.md5()
+    except ValueError:
+        m = hashlib.md5(usedforsecurity=False)
```

Comment:
Why dont we just pass `usedforsecurity=False` by default? Are there any implications?

Reply:
Yeah, in environments without FIPS throws this

```
Python 3.8.10 (default, Jun  2 2021, 10:49:15) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from hashlib import md5
>>> md5(usedforsecurity=False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'usedforsecurity' is an invalid keyword argument for openssl_md5()
```